srcpkgs/zoom/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'zoom'
+pkgname=zoom
+version=2.6.149990.1216
+revision=1
+wrksrc=$pkgname
+create_wrksrc=yes
+only_for_archs="i686 x86_64"
+short_desc="Video Conferencing and Web Conferencing Service"
+maintainer="Daniel Santana <daniel@santana.tech>"
+license="Proprietary"
+homepage="https://zoom.us/"
+distfiles="https://zoom.us/client/${version}/${pkgname}_${XBPS_TARGET_MACHINE}.pkg.tar.xz"
```

Comment:
This looks broken

Reply:
The checksum will vary depending on the architecture you're building for, so if you're building for x86_64 you will need checksum=foo but if you're building for i686 you will need checksum=bar

Reply:
And ofc that shows in the Travis, please check travis